to.etc.domui/src/main/java/to/etc/domui/component/upload/FileUpload2.java

```diff
@@ -407,5 +426,15 @@ public boolean handleUploadRequest(@NonNull RequestContextImpl param, @NonNull C
 	@Override public void setHint(String hintText) {
 		setTitle(hintText);
 	}
+
+	@Nullable
+	public IClicked<FileUpload2> getOnClearClicked() {
```

Comment:
Why can we not just have an onValueChanged handler? That should see when the value is cleared?